airflow/www/security.py

```diff
@@ -200,6 +200,16 @@ def __init__(self, appbuilder):
             view.datamodel = CustomSQLAInterface(view.datamodel.obj)
         self.perms = None
 
+    def _is_subdag(self, dag_id):
+        if '.' in dag_id:
+            dm = (
+                self.get_session.query(DagModel.dag_id, DagModel.is_subdag)
```

Comment:
```suggestion
                self.get_session.query(DagModel.is_subdag)
```

Only need this field, yeah?